backend/db/utils.py

```diff
@@ -688,4 +665,26 @@ def delete_codesets_from_db(codeset_ids):
     with get_db_connection() as con:
         code_sets_to_be_deleted = sql_query(
             con, f"""SELECT codeset_id, concept_set_name FROM code_sets WHERE id IN ({codeset_ids.sql_format()})"""
-        )
\ No newline at end of file
+        )
+
+
+def reset_temp_refresh_tables(schema: str = SCHEMA):
```

Comment:
## Resetting tables when error occurs
@Sigfried I think it would be nice if I had the time to set this up so that this exception handling occurred for all types of refreshes.

I've been wanting to do this for a while. Basically it is hella annoying, when I'm troubleshooting and in development, naturally errors occur, then I have to manually go and delete all the `_new` tables, and rename all the `_old` tables back to their original states. Finally automated that here.